### ISSUE TITLE
Support for imperial units?

### ISSUE THREAD

**Issue by sandeepmistry** [arduino-libraries/Arduino_MKRENV#3]
Suggested by @Akash73, we need a way to get the temperature in Fahrenheit instead of Celcius. Also, we should consider imperial units for the other sensors.

Here are some proposals:

1) Add an optional unit's parameter to API's, for example
```
ENV.readTemperature(); // defaults to celcius

ENV.readTemperature(FAHRENHEIT);

ENV.readTemperature(CELCIUS);
```

2) Add another API:
```
ENV.readTemperatureF();
```

@tigoe @cmaglie @facchinm @simonepda, any opinions on this?

**Comment by tigoe** [comment]
I prefer readTemperature(SCALE) where SCALE is FAHRENHEIT or CELSIUS, but I could take either one.

**Comment by akash73** [comment]
I agree with Tom

**Comment by Rocketct** [comment]
i'm agree with Tom!! if for you is ok i'll change immediatly the code @sandeepmistry

**Comment by Rocketct** [comment]
With @sandeepmistry , we have discus about add the support for millibar and kilopascal for the pressure sensor.
what do you think about?

For pressure:
ENV.readPressure(MILLIBAR)
ENV.readPressure(KILOPASCAL)

we should also add different scale for the light sensor?
like:
ENV.readIlluminance(CANDLE)
ENV.readIlluminance(LUX)

**Comment by sandeepmistry** [comment]
@akash73 @tigoe are you ok if we proceed with @Rocketct's proposal above?

**Comment by tigoe** [comment]
I am absolutely thrilled about it. Thank you.

**Comment by sandeepmistry** [comment]
Closed via @Rocketct's pull request #4.
